seth-sethgrosenberg/hw_02/3-dice-lab/js/app.js

```diff
@@ -0,0 +1,43 @@
+/*
+
+Creating a page where every time the page is loaded, the console logs random values for two dice. 
+
+1) Write down pseudocode for the program described below.
+
+when page is loaded
+
+dice 1 = random number between 1-6
+dice 2 = random number between 1-6
+
+print dice 1
+print dice 2
+
+2) Follow the steps in your pseudocode to write the code for your function.
+* generate a random number between 1 - 6 and store to a variable, firstDie
+* generate a random number between 1 - 6 and store to a variable, secondDie
+* log the values of the first and second dice to the console, labeling each value
+* log the total of the two dice to the console, with a label
+
+3) Create a statement that runs the function you created whenever the page loads.
+  (Hint: Reference when the page loads with the code window.onload, and set its value to the name of the function you created, without parentheses.)
+
+*/
+
+function randomIntFromInterval(min,max)
+{
```

Comment:
Did you drop this? `{`
Egyptian style brackets allow your code to determine where a block starts, and line placement matters.